python/ml_croissant/Makefile

```diff
@@ -0,0 +1,21 @@
+# flake8 enforces docstrings.
+flake8:
+	flake8 .
+
+pyflakes:
+	pyflakes .
+
+pylint:
+	pylint **/*.py
+
+pytest:
+	pytest .
+
+pytype:
+	pytype --jobs="$$(grep -c ^processor /proc/cpuinfo)" --config=./pyproject.toml
+
+# Launch all tests as launched on GitHub actions by order of importance.
```

Comment:
Why do we need to launch them by order of importance?

Reply:
Edit: I messed up and merged before pushing the changes: https://github.com/mlcommons/croissant/pull/172